src/Mealz/MealBundle/Service/SlotService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mealz\MealBundle\Service;
+
+use App\Mealz\MealBundle\Entity\Slot;
+use Doctrine\ORM\EntityManagerInterface;
+use InvalidArgumentException;
+
+class SlotService
+{
+    private EntityManagerInterface $em;
+
+    public function __construct(EntityManagerInterface $em)
+    {
+        $this->em = $em;
+    }
+
+    /**
+     * Update state (enabled/disabled) of a given slot.
```

Comment:
Ein bisschen unschön, da der Kommentar ja auf die interne Implementierung deutet. Also Kommentar bezieht sich ja auf die interne Implementierung der private Methode. Würde es entweder konkret machen "toggleSlot" oder allgemein (kann ja auch in Zukunft in $data der title geändert werden oder sowas) und den Kommentar entfernen.